src/OrchardCore.Modules/OrchardCore.Alias/Drivers/AliasPartDisplayDriver.cs

```diff
@@ -55,12 +57,12 @@ private async Task ValidateAsync(AliasPart alias, IUpdateModel updater)
         {
             if (alias.Alias?.Length > MaxAliasLength)
             {
-                updater.ModelState.AddModelError(Prefix, nameof(alias.Alias), S["Your alias is too long. The alias can only be up to {0} characters.", MaxAliasLength]);
+                updater.ModelState.AddModelError(Prefix, nameof(alias.Alias), S[DefaultMaxAliasLengthError, MaxAliasLength]);
```

Comment:
Can't use constants here or the PO localizer won't extract them. So keep the values the same, but as strings in the Validate Drivers